website/docs/cloud-docs/api-docs/no-code-provisioning.mdx

```diff
@@ -0,0 +1,491 @@
+              ]
+            }
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request PATCH \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/no-code-modules/nocode-9HE91XDNY3faePn2
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "nocode-9HE91XDNY3faePn2",
+    "type": "no-code-modules",
+    "attributes": {
+      "enabled": true,
+      "version-pin": "1.0.1"
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organizations"
+        },
+        "links": {
+          "related": "/api/v2/organizations/hashicorp"
+        }
+      },
+      "registry-module": {
+        "data": {
+          "id": "mod-2aaFrmRPZs2N9epr",
+          "type": "registry-modules"
+        },
+        "links": {
+          "related": "/api/v2/registry-modules/mod-2aaFrmRPZs2N9epr"
+        }
+      },
+      "variable-options": {
+        "data": [
+          {
+            "id": "ncvaropt-fcHDfnZ1EGdRzFNC",
+            "type": "variable-options"
+          },
+          {
+            "id": "ncvaropt-dZMfdh9KBcwFjyv2",
+            "type": "variable-options"
+          }
+        ]
+      }
+    },
+    "links": {
+      "self": "/api/v2/no-code-modules/nocode-9HE91XDNY3faePn2"
+    }
+  }
+}
+```
+
+## Read a No-Code-enabled module's properties
+
+
+`GET /no-code-modules/:id`
+
+
+Parameter | Description
+--- | ---
+`:id` | The unique identifier for a No-Code-Allowed module.
+
+Use this API to read the details of an existing No-Code-Allowed module.
+
+The unique identifier for the No-Code-Allowed module was returned in the response to [the API call that enabled the module for No-Code Provisioning](#allow-no-code-provisioning-of-a-module-for-an organization).
+
+| Status  | Response                                     | Reason                                                         |
+| ------- | -------------------------------------------- | -------------------------------------------------------------- |
+| [200][] | [JSON API document][] (`type: "no-code-modules"`) | Successfully read the no-code module.
+| [400][] | [JSON API error object][]                    | Invalid `include` parameter                                    |
+| [404][] | [JSON API error object][]                    | Not found, or user unauthorized to perform action |
+| [422][] | [JSON API error object][]                    | Malformed request body (missing attributes, wrong types, etc.) |
+| [500][] | [JSON API error object][]                    | Internal system failure                                   |
+
+### Query Parameters
+
+[These are standard URL query parameters](/terraform/cloud-docs/api-docs#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
```

Comment:
Adding a suggestion for clarity in the first sentence:
```suggestion
[This endpoint uses our standard URL query parameters](/terraform/cloud-docs/api-docs#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
```

Reply:
Your suggestion does improve the clarity here. FYI this sentence is taken verbatim from the `_template.mdx` file in this folder, which I used as the framework for structuring the docs consistently. Maybe that template ought to be adjusted?